tests/unit/utils.py

```diff
@@ -0,0 +1,20 @@
+from dusty.schemas.base_schema_class import DustySchema
+from dusty.schemas.app_schema import app_schema
+from dusty.schemas.lib_schema import lib_schema
+from dusty.schemas.bundle_schema import bundle_schema
+
+
+def get_app_dusty_schema(doc):
+    if 'image' not in doc and 'build' not in doc:
```

Comment:
This is in testing utils.  The idea is that I want to shelter those writing tests from needing to remeber all of the required fields.  So if they do not pass them in, i provide a default.